src/styles/main.scss

```diff
@@ -1,3 +1,67 @@
+$baza: 40px;
+
 body {
   margin: 0;
+  padding: 20px;
+  box-sizing: border-box;
+}
+
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 730px;
+  
+  & :first-child {
+    margin-left: 622px;
+  }
+
+  @for $index from 1 through 31 {
+    & :nth-child(#{$index})::before {
+      position: absolute;
+      content: "#{$index}";
+      width: 20px ;
+      height: 20px;
+      left: $baza;
+      top: $baza;
+      font-family: Arial, sans-serif;
+      font-size: 22px;
+      color: black;
+    }
+  }
+
+  &__cell {
+    position: relative;
+    width: 100px;
+    height: 100px;
+    margin: 1px;
+    border: 1px solid black;
+    background-color: #eee;
+    cursor: pointer;
+    transition: 0.5s;
+ 
+    &:hover {
+      background-color: pink;
+    }
+
+    &:active {
+      transition: 0.5s;
+      transform: translateY(-20px);
+    }
+  }
+  
+  @for $i from 1 through 3 {
+    &__cell--month-length--#{(28+$i)} {
+      display: none;
+    }
+  }
+  
+  $days: 
+  mon, tue, wed, thu, fri, sat, sum;
+
+  @each $day in $days {
+    &__cell--start-day--#{$day} {
+      background-color: #bbb;
+    }
+  }
```

Comment:
something similar is here. the `calendar` class has an additional modifier as `calendar--start-day--{day}`. it means the calendar should start from a specific day (e.g. `calendar--start-day--fri` starts with Friday). I would recommend you add `margin-left` (as said in the task) to move the first day a little bit.

If you need any help or have questions - feel free to ask in chat or join to the next HTML/CSS Q&A session